crates/bindings/src/table.rs

```diff
@@ -384,10 +449,75 @@ impl<Tbl: Table, IndexType, Idx: Index> BTreeIndex<Tbl, IndexType, Idx> {
 
     /// Returns an iterator over all rows in the database state where the indexed column(s) match the bounds `b`.
     ///
-    /// `b` may be:
+    /// This method accepts a variable numbers of arguments using the [`BTreeIndexBounds`] trait.
+    /// This depends on the type of the B-Tree index. `b` may be:
     /// - A value for the first indexed column.
     /// - A range of values for the first indexed column.
     /// - A tuple of values for any prefix of the indexed columns, optionally terminated by a range for the next.
+    ///
+    /// For example:
+    ///
+    /// ```no_run
+    /// #[table(name = users,
```

Comment:
I'm gonna stop leaving these comments. You should just grep for `name =` and fix them all.